engine/src/main/java/io/camunda/zeebe/engine/processing/timer/TriggerTimerProcessor.java

```diff
@@ -110,7 +110,7 @@ public void processRecord(
           processDefinitionKey, processInstanceKey, timer.getTargetElementIdBuffer(), NO_VARIABLES);
     } else {
       final var elementInstance = elementInstanceState.getInstance(elementInstanceKey);
-      if (!eventHandle.canTriggerElement(elementInstance)) {
+      if (!eventHandle.canTriggerElement(elementInstance, timer.getTargetElementIdBuffer())) {
```

Comment:
:question: What is the difference between `eventHandle.canTriggerElement` and `eventScopeInstance.canTriggerEvent`? Is there a change to harmonize method names?

Reply:
I see your point. `EventHandle` uses the term `element` (because not all elements are events like receive task) but `EventScopeState` uses `event`. It would make sense to align the naming. But let's postpone it because it would require more renaming.